x/ccv/provider/keeper/throttle.go

```diff
@@ -272,6 +277,7 @@ func (k Keeper) SetThrottledPacketDataSize(ctx sdktypes.Context, consumerChainID
 // queue for the given consumer chain.
 func (k Keeper) IncrementThrottledPacketDataSize(ctx sdktypes.Context, consumerChainID string) {
 	size := k.GetThrottledPacketDataSize(ctx, consumerChainID)
+	k.Logger(ctx).Debug("increment throttled packets size", "chain-id", consumerChainID, "size", size+1)
```

Comment:
same on moving this to after the setter
```suggestion
	k.Logger(ctx).Debug("incremented throttled packets size", "chain-id", consumerChainID, "size", size+1)
```